experimental/harfbuzz/src/lib.rs

```diff
@@ -198,6 +200,31 @@ unsafe extern "C" fn icu4x_hb_unicode_decompose_destroy(user_data: *mut c_void)
     let _ = Box::from_raw(user_data as *mut CanonicalDecomposition);
 }
 
+/// Returns the Bidi_Mirroring_Glyph, but adjusting the return value
+/// to fix HarfBuzz expected behavior for code points whose property value
+/// for Bidi_Mirroring_Glyph is the undefined value.
+///
+/// From HarfBuzz docs on `hb_unicode_mirroring_func_t`:
+/// <note>Note: If a code point does not have a specified
+/// Bi-Directional Mirroring Glyph defined, the method should
+/// return the original code point.</note>
+unsafe extern "C" fn icu4x_hb_unicode_mirroring(
```

Comment:
Nit: Keep these functions in order:

1. CCC
2. GC
3. Bidi Mirroring
4. Script
5. Composition
6. Decomposition

Reply:
Done.